Cargo.toml

```diff
@@ -17,8 +21,17 @@ object = { version = "0.29.0", default-features = false, features = ['archive']
 [lib]
 crate-type = ["cdylib"]
 
+[profile.release]
+# Omit any unwinding support.
+panic = 'abort'
+# Omit integer overflow checks, which include failure messages which require
+# string initializers.
+overflow-checks = false
```

Comment:
I think this is the default for `release` mode and doesn't need to be specified?